### DISCUSSION TITLE
Customise Autocomplete Dropdown

### DISCUSSION THREAD

**Discussion by mchandlernz** [mantinedev/mantine#968] [Q&A] [answered]
Hello, love the library so far! 

I was wondering if it was possible to customise the Autocomplete dropdown component?

I had a use case where I wanted to group the results based on their categories and add headers to the category.

Kind of like this:

![Screen Shot 2022-03-10 at 4 56 33 PM](https://user-images.githubusercontent.com/50541874/157585655-e6a8f866-90a2-4bb2-8c13-1d38844c2c0a.png)

**Comment by rtivital** [comment, marked as answer]
You can group Autocomplete items the same was as in Select component – https://codesandbox.io/s/vigilant-bas-nx9u30?file=/src/App.tsx
